documentation/modules/serving/pages/knative-client.adoc

```diff
@@ -60,7 +60,6 @@ To create the `greeter` service using `kn` run the following command:
 [source,bash,subs="+macros,attributes+"]
 ----
 kn service create greeter \
-  --namespace {tutorial-namespace} \
```

Comment:
Commands in this page don't have `--namespace` in them. I removed 1-2 I saw.

Reply:
I think its better to add namespace everywhere for consistency with other chapters ? Can you fix that ?